### PR TITLE
cyber:support to copy softlink file

### PR DESCRIPTION
**Pull request by gengqx**

### CODE REVIEW

File: cyber/common/file.cc [hunk 1]
```diff
@@ -163,8 +163,17 @@ std::vector<std::string> Glob(const std::string &pattern) {
...
   std::ifstream src(from, std::ios::binary);
   if (!src) {
-    AERROR << "Source path doesn't exist: " << from;
-    return false;
+    AWARN << "Source path could not be normally opened: " << from;
+    std::string command = "cp -r " + from + " " + to;
+    ADEBUG << command;
+    const int ret = std::system(command.c_str());
+    if (ret == 0) {
+      ADEBUG << "Copy Success,Command returns " << ret;
```
Comment (xiaoxq) [inline cyber/common/file.cc]: Refine the sentence: `"Copy success, command returns "`

File: cyber/common/file.cc [hunk 2]
```diff
@@ -163,8 +163,17 @@ std::vector<std::string> Glob(const std::string &pattern) {
...
+      return true;
+    } else {
+      ADEBUG << "Copy Error,Command returns " << ret;
```
Comment (xiaoxq) [inline cyber/common/file.cc]: ditto

File: cyber/common/file.cc (hunk 1 above)
Comment (gengqx) [reply cyber/common/file.cc]: Done

File: cyber/common/file.cc (hunk 2 above)
Comment (gengqx) [reply cyber/common/file.cc]: Done

**Pull request merged**